__tests__/reviews.test.js

```diff
@@ -206,4 +206,106 @@ describe("GET /api/reviews", () =>{
             })
         })
     })
+
+    test("status 200: results sorted by title descending", () =>{
```

Comment:
I don't think you need to write a test for all of the possible sort_by options. Probably just one or two that are valid, and then one that isn't.